Evaluate-ModelarDB-Compression/main.py

```diff
@@ -342,12 +342,17 @@ def send_sigint_to_process(process):
                 try:
                     test_data_field_column = test_data.column(field_column_name)
                 except:
-                    # Spaces in the name may have been replaced by underscores.
-                    field_column_name_with_space = field_column_name.replace("_", " ")
-                    test_data_field_column = test_data.column(
-                        field_column_name_with_space
-                    )
-
+                    if field_column_name.__contains__("_"):
+                        # Spaces in the name may have been replaced by underscores.
+                        field_column_name_with_space = field_column_name.replace("_", " ")
+                        test_data_field_column = test_data.column(
+                            field_column_name_with_space
+                        )
+                    else: 
+                        # Upper case columns may have been converted to lower case.
+                        test_data_cols = [i.lower() for i in test_data.column_names]
```

Comment:
Why convert all column names to lowercase and then only use one of them? Seems this could be replaced with `field_column_name_lowercase_with_space = field_column_name_with_space.lower()` between Line 347 and Line 348 and than changing `field_column_name_with_space` with `field_column_name_lowercase_with_space`. There is no reason to make a complex solution if a simple solution suffices.

Reply:
fixed.